### PR TITLE
Update the 'add named arg' refactoring to support only adding to the current argument (for C# 7.2 and above).

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Note: we will still want to eventually convert htis to an analyzer, along with a user preference for things like "prefer named arguments for literals" in the future.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @roslyn-ide @jcouv

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by jcouv** [comment]
@CyrusNajmabadi Nice :-)
FYI I started working on code style (prefer naming literal arguments).

**Comment by jcouv** [comment]
@CyrusNajmabadi From discussion with DavidP, it sounds like we should keep this refactoring, and then add an analyzer/codefixer pair for the code style. Unfortunately, I didn't see a great way of sharing code. Let me know if you have an idea.

File: src/Features/VisualBasic/Portable/CodeRefactorings/UseNamedArguments/VisualBasicUseNamedArgumentsCodeRefactoringProvider.vb [hunk 1]
```diff
@@ -47,6 +47,10 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.CodeRefactorings.UseNamedArguments
             Protected Overrides Function IsCloseParenOrComma(token As SyntaxToken) As Boolean
                 Return token.IsKind(SyntaxKind.CloseParenToken, SyntaxKind.CommaToken)
             End Function
+
+            Protected Overrides Function SupportsNonTrailingNamedArguments(options As ParseOptions) As Boolean
+                Return False
```
Comment (jcouv) [inline src/Features/VisualBasic/Portable/CodeRefactorings/UseNamedArguments/VisualBasicUseNamedArgumentsCodeRefactoringProvider.vb]: VB 15.5 does support non-trailing named arguments too. Next PR? ;-)

Comment (CyrusNajmabadi) [reply src/Features/VisualBasic/Portable/CodeRefactorings/UseNamedArguments/VisualBasicUseNamedArgumentsCodeRefactoringProvider.vb]: oh, it does?  i didn't know you did that :)

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by CyrusNajmabadi** [comment]
retest windows_debug_unit64_prtest please

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-ide @pilchie this is ready for merge.  tnx.

**Comment by CyrusNajmabadi** [comment]
@dotnet/roslyn-ide @Pilchie this is ready for merge. tnx.

File: src/Features/Core/Portable/FeaturesResources.resx [hunk 2]
```diff
@@ -1307,7 +1307,7 @@ This version used in: {2}</value>
   <data name="Switching_between_lambda_and_local_function_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
     <value>Switching between a lambda and a local function will prevent the debug session from continuing.</value>
   </data>
-  <data name="Switching_between_lambda_and_local_function_will_prevent_the_debug_session_from_continuing" xml:space="preserve">
```
Comment (rchande) [inline src/Features/Core/Portable/FeaturesResources.resx]: Was this removed intentionally? Seems unrelated...

Comment (jcouv) [reply src/Features/Core/Portable/FeaturesResources.resx]: It was duplicate of one above.

---
In reply to: [138757442](https://github.com/dotnet/roslyn/pull/21980#discussion_r138757442) [](ancestors = 138757442)
